library/core/src/iter/traits/iterator.rs

```diff
@@ -107,8 +107,6 @@ pub trait Iterator {
     /// again may or may not eventually start returning [`Some(Item)`] again at some
     /// point.
     ///
-    /// [`Some(Item)`]: Some
-    ///
```

Comment:
This will silently break: https://github.com/rust-lang/rust/pull/75969#issuecomment-683422039
```suggestion
/// [`Some(Item)`]: Some
///